src/components/z-divider/styles.css

```diff
@@ -0,0 +1,37 @@
+:host {
+  display: block;
+  padding: 0;
+  margin: 0;
+}
+
+:host.divider-horizontal {
+  width: 100%;
+}
+
+:host.divider-vertical {
+  height: 100%;
+}
+
+:host.divider-small.divider-horizontal {
+  height: 1px;
```

Comment:
I think this sizes are related to the ones of the borders, so we can use the related tokens (`--border-size-small`, medium and large)

Reply:
Risolto